internal/pkg/skuba/addons/kured_test.go

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package addons
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/SUSE/skuba/internal/pkg/skuba/kubernetes"
+	"k8s.io/apimachinery/pkg/util/version"
+)
+
+func TestGetKuredImage(t *testing.T) {
+	type args struct {
+		imageTag string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		// TODO: Add test cases.
+		{
+			name: "TestGetKuredImage",
+			args: args{
+				imageTag: "1.2.0-rev4",
+			},
+			want: "registry.suse.de/devel/caasp/4.0/containers/containers/caasp/v4/kured:1.2.0-rev4",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := GetKuredImage(tt.args.imageTag); got != tt.want {
```

Comment:
Suggest to use regexp.

Reply:
this function expects exact output from args.